src/STAT.md

```diff
@@ -1,5 +1,17 @@
 # LCD Status Register
 
+## FF44 — LY: LCD Y coordinate \[read-only\]
```

Comment:
I think in any case they should be after the terminology box, as it's meant to be an introductory block or text to satisfy so me knowledge pre-requirements